roles/matrix-postgres/templates/init-additional-db-user-and-role.sql.j2

```diff
@@ -0,0 +1,19 @@
+-- `CREATE USER` does not support `IF NOT EXISTS`, so we use this workaround to prevent an error and raise a notice instead.
+-- Seen here: https://stackoverflow.com/a/49858797
+DO $$
+BEGIN
+  CREATE USER {{ additional_db.username }};
+  EXCEPTION WHEN DUPLICATE_OBJECT THEN
+  RAISE NOTICE 'not creating role {{ additional_db.username }}, since it already exists';
```

Comment:
'not creating user `{{ additional_db.username }}` ...'? - the `role` is (imo) a bit confusing, does postgres really call their users "role", or do the users _have_ roles?

Reply:
Postgres seems to call these roles, yes. And some roles have login support (`CREATE ROLE .. WITH LOGIN`) while others are just about data ownership, but don't provide login support (`CREATE ROLE ... WITH NOLOGIN`).

`CREATE USER` is a shortcut for saying `CREATE ROLE .. WITH LOGIN`.

I've fixed this notice message in f1e85f7112ffa9dd